plugins/native/module/vtk.module

```diff
@@ -5,6 +5,7 @@ DESCRIPTION
 DEPENDS
   VTK::CommonCore
   VTK::CommonExecutionModel
+  f3d::vtkext
```

Comment:
i dont think this is needed ?

Reply:
It's used to compile the importer and unit test.